Source/Plugins/Core/com.equella.core/js/tsrc/mainui/routes.tsx

```diff
@@ -37,6 +37,7 @@ export interface OEQRouteComponentProps<T = any>
   redirect(to: LocationDescriptor): void;
   setPreventNavigation(b: boolean): void;
   refreshUser(): void;
+  reloadNeeded: boolean;
```

Comment:
a convention I've seen elsewhere, prefixing flags with `is` or `are` to differentiate them from actions.
In this case it could look something like:

```suggestion
  isReloadNeeded: boolean;
```